cmd/tealdbg/localLedger.go

```diff
@@ -53,6 +69,25 @@ func makeAppLedger(
 	apps := []basics.AppIndex{appIdx}
 	apps = append(apps, txn.Txn.ForeignApps...)
 
+	// first populate balances from the indexer
+	for _, acc := range accounts {
+		fmt.Println("debug hello")
+		if _, ok := balances[acc]; !ok {
+			queryString := fmt.Sprintf("%s/v2/accounts/%s?round=%v", indexerURL, acc, round)
```

Comment:
Consider using `path.Join` here (from https://golang.org/pkg/path/)

Reply:
can path.join handle the `?round=x` part?